librocksdb_sys/src/lib.rs

```diff
@@ -976,6 +980,46 @@ extern "C" {
                                                       props: *mut DBTablePropertiesCollection,
                                                       errptr: *mut *mut c_char);
 
+    // Get All Key Versions
+    pub fn crocksdb_keyversions_create() -> *mut DBKeyVersions;
+
+    pub fn crocksdb_keyversions_destroy(kvs: *mut DBKeyVersions);
+
+    pub fn crocksdb_keyversions_iterator_create(kv: *mut DBKeyVersions)
```

Comment:
or you can add a get_count function like c_rocksdb_keyversions_count(kvs: *DBKeyVersions) to get count first.